### PR TITLE
sort chart 2

### PR DESCRIPTION
**Pull request by anthonyettinger**
- fix svg size
- make sure d3 is not overriden

@jtary @seanbrookes PTAL

### CODE REVIEW

File: client/www/index.html [hunk 1]
```diff
@@ -69,6 +69,7 @@
 <script src="./scripts/vendor/react/react-with-addons.js"></script>
 
 <!-- d3 -->
+<script src="./scripts/modules/tracing/tracing.viz.module.js"></script>
```
Comment (seanbrookes) [inline client/www/index.html]: why is this file moved here?  It depends on d3 so should be loaded after d3.js.  Plus it should be with the tracing code.

**Comment by anthonyettinger** [comment]
test please

File: client/www/index.html (hunk 1 above)
Comment (anthonyettinger) [inline client/www/index.html]: This is just a quick hack so the `d3` object doesn't get override. as we discussed you will test a better solution w/ tracing in your env.
